src/pages/iou/IOUTransactions.js

```diff
@@ -16,10 +16,10 @@ const propTypes = {
     reportActions: PropTypes.shape(reportActionPropTypes),
 
     /** ReportID for the associated chat report */
-    chatReportID: PropTypes.number.isRequired,
+    chatReportID: PropTypes.string.isRequired,
 
     /** ReportID for the associated IOU report */
-    iouReportID: PropTypes.number.isRequired,
+    iouReportID: PropTypes.string.isRequired,
```

Comment:
Since this is string, you will need to make the this.props.iouReportID number on the line 58 below, at the moment this is causing the cancel button is not showing for the requests.